cmd/algocfg/getCommand_test.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2019-2022 Algorand, Inc.
+// This file is part of go-algorand
+//
+// go-algorand is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// go-algorand is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPrint(t *testing.T) {
+	testcases := []struct {
+		input    interface{}
+		expected string
+	}{
+		{
+			input:    uint64(1234),
+			expected: "1234",
+		},
+		{
+			input:    int64(-1234),
+			expected: "-1234",
+		},
+		{
+			input:    true,
+			expected: "true",
+		},
+		{
+			input:    time.Second,
+			expected: "1s",
+		},
+	}
+	for i, tc := range testcases {
+		t.Run(fmt.Sprintf("test %d", i), func(t *testing.T) {
+			var buf bytes.Buffer
```

Comment:
and around here `val, err := getObjectProperty(tc, "Input")`
Then Fprintf or Sprintf("%v", val)

Reply:
Using the test case as an input to the test function is genius.